_includes/v23.1/ui/databases.md

```diff
@@ -71,6 +75,8 @@ The table details include:
 - **Auto Stats Collection**: Whether [automatic statistics collection]({{ link_prefix }}cost-based-optimizer.html#table-statistics) is enabled.
 {% if page.cloud != true %}
 - **Regions/Nodes**: The regions and nodes on which the table data is stored. This is not displayed on a single-node cluster.
+{% else %}
+- **Regions**: The regions on which the table data is stored.
```

Comment:
At this point I wonder if we should reuse this fragment to keep it in sync more easily across 4 different lines.